### PR TITLE
docs(AwsVpcPeering): documentation

### PR DESCRIPTION
**Pull request by vandjelk**
**Description**

Changes proposed in this pull request:

- AWS VpcPeering documentation & tutorial

### CODE REVIEW

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Kyma and the remote AWS Virtual Private Cloud (VPC) network. Virtual network peering is only possible within the networks
+of the same cloud provider.
+
+Once an AwsVpcPeering CR is created and reconciled, the Cloud Manager controller first creates a virtual network peering
+connection in the Virtual Private Cloud (VPC) network of the Kyma cluster in the underlying cloud provider and accepts
+VPC peering connection in the remote cloud provider account.
+
+You must create CloudManagerPeeringRole and authorize Cloud Manager principal to assume that role in the remote cloud provider 
+account to accept VPC peering connection. Assign the following permissions to CloudManagerPeeringRole in the 
+remote account:
```
Comment (dushanpantic) [inline docs/user/resources/04-70-10-aws-vpc-peering.md:14]: 😵

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* ec2:AcceptVpcPeeringConnection
+* ec2:DescribeVpcs
+* ec2:DescribeVpcPeeringConnections
+* ec2:DescribeRouteTables
+* ec2:CreateRoute
+* ec2:CreateTags
+
+AwsVpcPeering can be deleted at any time but the VPC peering connection in the remote account must be deleted
+manually.
```
Comment (dushanpantic) [inline docs/user/resources/04-70-10-aws-vpc-peering.md:23]: In my opinion, this should be written in a way to reflect that the kyma-side cloud provider peering is deleted as part of AwsVpcPeering deletion, but remote one is left hanging, and must be deleted manually.

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+This table lists the parameters of the given resource together with their descriptions:
+
+**Spec:**
+
+| Parameter           | Type   | Description                                                                        |
+|---------------------|--------|------------------------------------------------------------------------------------|
+| **remoteAccountId** | string | Specifies the the Amazon Web Services account ID of the owner of the accepter VPC. |
+| **remoteRegion**    | string | Specifies the Region code for the accepter VPC.                                    |
+| **remoteVpcId**     | string | Specifies the ID of the VPC with which you are creating the VPC peering connection |
```
Comment (dushanpantic) [inline docs/user/resources/04-70-10-aws-vpc-peering.md:36]: Please specify if params are `Required` or `Optional`

e.g. 
```
| **remoteAccountId** | string | Required. Specifies the the Amazon Web Services account ID of the owner of the accepter VPC. 
```

**Review by dushanpantic — DISMISSED** [review]

File: docs/user/tutorials/01-40-aws-vpc-peering.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+    export NODE_NETWORK=$(kubectl get cm -n kube-system shoot-info -o jsonpath='{.data.nodeNetwork}')
+    export VPC_NAME=my-vpc
+    export VPC_ID=$(aws ec2 create-vpc --cidr-block $CIDR_BLOCK --tag-specifications ResourceType=vpc,Tags=[{Key=$SHOOT_NAME,Value=""},{Key=Name,Value=$VPC_NAME}] --query Vpc.VpcId --output text)  
+    ```
+8.  Create subnet
+    ```shell
+    export SUBNET_ID=$(aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block $CIDR_BLOCK --query Subnet.SubnetId --output text) 
+    ```
+
+9.  Run instance
```
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply docs/user/tutorials/01-40-aws-vpc-peering.md:85]: I assume this is about a specific (user's) instance, so I'd suggest you use `the` before it. If that's not the case, please use `an instance`.

File: docs/user/tutorials/01-40-aws-vpc-peering.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+This tutorial explains how to create a VPC peering connection between a remote VPC network and Kyma in AWS. The tutorial
+assumes that the Cloud Manager module is enabled in your Kyma cluster. Follow the steps from this tutorial to create a 
+new VPC network, and VM, and assign required permissions to the provided Kyma account and role in your AWS account. If you want to
+use the existing resources instead of creating new ones, adjust variable names accordingly and skip the steps that 
+create those resources.
+
+## Steps <!-- {docsify-ignore} -->
+
+1.  Set default AWS CLI profile. If you haven't configured the profile it yet please go see AWS documentation https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html
```
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/tutorials/01-40-aws-vpc-peering.md:11]: Yes. It makes sense that Configuration of the default AWS CLI profile is the prerequisite.

Should I introduce Prerequisite section above Steps?

File: docs/user/tutorials/01-40-aws-vpc-peering.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+        ]
+    }
+    EOF
+    ```
+3. Create VpcPeeringRole and attach a trust policy to document.
+    ```shell
+    export AWS_ROLE_NAME=CloudManagerPeeringRole
+    aws iam create-role --role-name $AWS_ROLE_NAME --assume-role-policy-document file://./trust_policy.json 
+    ```
+4.  Create policy document that will be used to create policy
```
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/tutorials/01-40-aws-vpc-peering.md:41]: There are two policies. 

- Trust policy defining entities that can assume the role under specified conditions.
- Permissions policy defining which actions are allowed or denied to the role.

File: docs/user/tutorials/01-40-aws-vpc-peering.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/tutorials/01-40-aws-vpc-peering.md:85]: It is more like run a virtual machine instance but in the scope of AWS it is just an instance. So perhaps `an instance` would be more appropriate.

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+	]
+}
+```
+Create new managed policy CloudManagerPeeringAccess with following permissions:
+* ec2:AcceptVpcPeeringConnection
+* ec2:DescribeVpcs
+* ec2:DescribeVpcPeeringConnections
+* ec2:DescribeRouteTables
+* ec2:CreateRoute
+* ec2:CreateTags
```
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/resources/04-70-10-aws-vpc-peering.md]: I reformatted this into JSON code as it is better fit for AWS Policy.

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
+## AWS Vpc Peering
+
+
```
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/resources/04-70-10-aws-vpc-peering.md]: I have added subsections to add more clarity. Let me know if you have more suggestions.

File: docs/user/resources/04-70-10-aws-vpc-peering.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
+## AWS VPC Peering
```
[the remark this answers is not in the document]
Comment (vandjelk) [reply docs/user/resources/04-70-10-aws-vpc-peering.md]: It's a title, and it should be heading 1. I corrected it for docs/user/resources/04-40-10-azure-vpc-peering.md as well.  Thanks

**Review by IwonaLanger — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**